### ISSUE TITLE
plotCorrelation - ValueError: rotation must be 'vertical', 'horizontal' or a number, not 45

### ISSUE THREAD

**Issue by gremame** [deeptools/deepTools#1256]
I tried the `plotCorrelation` function using different outputs generated by `multiBamSummary`. I get the error below when setting `--whatToPlot scatterplot`. The execution finishes successfully if this parameter is set to `heatmap`.

**Command I am using:**

```
plotCorrelation -in coverages.npz --corMethod pearson --whatToPlot scatterplot --removeOutliers  --skipZeros  --plotNumbers  --plotTitle "Correlation Plot" --colorMap RdYlBu -o correlation.plot.pdf --plotFileFormat pdf  --outFileCorMatrix correlation.matrix.txt
```

**Error I am getting:**
```
Traceback (most recent call last):
  File "/usr/local/bin/plotCorrelation", line 12, in <module>
    main(args)
  File "/usr/local/lib/python3.10/site-packages/deeptools/plotCorrelation.py", line 239, in main
    corr.plot_scatter(args.plotFile,
  File "/usr/local/lib/python3.10/site-packages/deeptools/correlation.py", line 507, in plot_scatter
    tick.label.set_rotation('45')
  File "/usr/local/lib/python3.10/site-packages/matplotlib/text.py", line 1196, in set_rotation
    raise ValueError("rotation must be 'vertical', 'horizontal' or "
ValueError: rotation must be 'vertical', 'horizontal' or a number, not 45
```

**OS and others**
```
Debian GNU/Linux 11 (bullseye)
Python 3.10.3
deeptools 3.5.1
```

**Comment by gremame** [comment]
OK, I found out why this was failing. There is a bug in the `correlation.py` when calling the `tick.label.set_rotation` function. The script is passing a character but the method expects a number. There are two calls to this method in the script. After fixing those, the program ran successfully.

**Comment by gremame** [comment]
I see you have this fixed in version 3.5.3
So nothing to do!
